base/src/main/java/com/smartdevicelink/proxy/rpc/GetVehicleData.java

```diff
@@ -1,828 +1,828 @@
-/*
```

Comment:
@kostyaBoss It looks like the entire file `GetVehicleData.java` is being regenerated by the code generator tool.  Can you just merge in the changes for this PR (data related to `seatOccupancy`)?